docker_build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly IMAGE_NAME="jellyninjas/terminalfighter"
+
+if [[ $# -eq 0 ]]; then
+	echo "usage: docker_build [init | build | <build.sh args>]"
```

Comment:
I don't really understand the `build` option... 

I just run 

`./docker_build docs`
or
`./docker_build src`

But running the "build" option gives me the following:
```
./docker_build build
usage: ./build.sh [all | src | format | docs | clean]
```

And giving it no options is kinda confusing too:

```
sudo ./docker_build.sh
usage: docker_build [init | build | <build.sh args>]
usage: ./build.sh [all | src | format | docs | clean]
```

Reply:
The build option wasn't supposed to be there, usage statement is updated. 

```
	echo "usage: docker_build [init | shell | <build.sh args>]"
```